src/store/api.ts

```diff
@@ -0,0 +1,38 @@
+import { initFirestore } from '../firestore'
+import {
+  collection,
+  addDoc,
+  updateDoc,
+  arrayUnion,
+  DocumentReference
+} from 'firebase/firestore'
+import { USER_COLLECTION_NAME } from '../const'
+import { writable } from 'svelte/store'
```

Comment:
Move `import` from packages to the top of the file.

Reply:
✅